packages/oauth/jwk/src/jwk.ts

```diff
@@ -122,19 +122,20 @@ export const jwkSchema = z.union([
 
 export type Jwk = z.infer<typeof jwkSchema>
 
-export const jwkValidator = jwkSchema
-  .refine((k) => k.use != null || k.key_ops != null, 'use or key_ops required')
-  .refine(
-    (k) =>
-      !k.use ||
-      !k.key_ops ||
-      k.key_ops.every((o) =>
-        k.use === 'sig'
-          ? o === 'sign' || o === 'verify'
-          : o === 'encrypt' || o === 'decrypt',
-      ),
-    'use and key_ops must be consistent',
-  )
+export const jwkValidator = jwkSchema.refine(
+  (k) =>
+    !k.use ||
+    !k.key_ops ||
+    k.key_ops.every(
+      k.use === 'sig'
+        ? (o) => o === 'sign' || o === 'verify'
+        : (o) => o === 'encrypt' || o === 'decrypt',
+    ),
```

Comment:
With the caveat that I'm unfamiliar with zod, this looks like it does what we want.

Although, `sig` is only for public keys - is it "consistent" to allow the `key_ops` values of "sign" and "encrypt"?

("consistent" as required at the end of https://datatracker.ietf.org/doc/html/rfc7517#section-4.3 )